sound_manager.py

```
@@ -0,0 +1,64 @@
+from arcade import Sound, load_sound
+
+DEFAULT_VOLUME = 0.7
+SFXPATH_BEETLE_HIT = "Assets/Sound/SFX/impact2_noisecollector.wav"
+SFXPATH_BEETLE_DEAD = "Assets/Sound/SFX/beetle_dead_mixkit.wav"
+SFX_PATH_PEASHOOTER = "Assets\Sound\SFX\pew_xtrgamr.wav"
+
+class SoundManager:
+    def __init__(self):
+        self.audio_player = Sound()
+        self.sfx_beetle_dead = load_sound(SFXPATH_BEETLE_DEAD)
+        self.sfx_beetle_hit = load_sound(SFXPATH_BEETLE_HIT)
+        self.peashooter_pew = load_sound(SFX_PATH_PEASHOOTER)
+
+    def setup(self):
+        self.set_volume(DEFAULT_VOLUME)
+
+    def set_volume(self, volume=None):
+        if not volume:
+            self.audio_player.set_volume = 0.7
+        else:
+            self.audio_player.set_volume = volume
+
+    def play_sound(self, sound):
+        if not sound:
+            pass
```

Might be worth throwing an exception here instead of just swallowing it.